google/cloud/bigtable/internal/common_client.h

```diff
@@ -112,6 +112,15 @@ class CommonClient {
       channels.swap(channels_);
       tmp.swap(stubs_);
       current_index_ = 0;
+    } else {
+      // Some other thread created the pool and saved it in `stubs_`. The work
+      // in this thread was superfluous. We release the lock while clearing the
+      // channels to minimize contention, this seems to workaround other bugs
```

Comment:
s/contention, this/contention. This/

Although, I'm not sure the "seems to workaround" part is useful.  I'm not even sure what "the Google implementation of std::mutex" is referring to.

Reply:
I'm still not sure what "bugs inside Google" means, but you can fill me in later.